server/parsers/basic_input_output_parser.ts

```diff
@@ -13,19 +13,79 @@ const sanitize = (content: string) => {
   return DOMPurify.sanitize(content, { FORBID_TAGS: ['img'] }).replace(/!+\[/g, '[');
 };
 
+const isEveryItemString = (array?: unknown) =>
+  !!(Array.isArray(array) && array.every((item) => typeof item === 'string'));
+
+export const parseSuggestedActions = (string: string): string[] => {
+  if (!string) {
+    return [];
+  }
+  const suggestedOutputString = string;
+  let suggestedActions: string[] = [];
+  try {
+    suggestedActions = JSON.parse(suggestedOutputString || '[]');
+  } catch (e) {
+    suggestedActions = [];
+  }
+
+  if (suggestedActions.length) {
+    return suggestedActions;
```

Comment:
Shall we check every item is string string here?

Reply:
Yeah, we should.

Reply:
Done